autopush-common/src/db/bigtable/bigtable_client/mod.rs

```diff
@@ -1853,16 +1868,17 @@ mod tests {
             panic!("Expected row");
         };
 
-        let expiry2 = row.take_required_cell("connected_at").unwrap().timestamp;
-        assert!(expiry2 > expiry);
+        let ca_expiry2 = row.take_required_cell("connected_at").unwrap().timestamp;
+
+        assert!(ca_expiry2 >= ca_expiry);
```

Comment:
I think this should still be `>` to ensure it was touched. If it's causing you any flake let's just add a small sleep instead.
```suggestion
        assert!(ca_expiry2 > ca_expiry);
```

Reply:
It was, I'll add a sleep.